core/src/main/scala/cats/std/either.scala

```diff
@@ -23,7 +23,7 @@ trait EitherInstances extends EitherInstances1 {
         }
     }
 
-  implicit def eitherInstances[A]: Monad[Either[A, ?]] with Traverse[Either[A, ?]] =
+  implicit def catsMonadForEither[A]: Monad[Either[A, ?]] with Traverse[Either[A, ?]] =
```

Comment:
As per https://github.com/typelevel/cats/issues/1061#issuecomment-221559248, perhaps the full package name should be incuded? So here, `cats` => `catsStd`. An then `catsKernel`, `catsKernelStd` and so on